packages/nextjs/components/Mapbox.js

```diff
@@ -1,18 +1,21 @@
 // eslint-disable-line import/no-webpack-loader-syntax
 import mapboxgl from "!mapbox-gl";
+import { blo } from "blo";
```

Comment:
Great placeholder. 

Next step would probably be to use the [Airstack API](https://docs.airstack.xyz/airstack-docs-and-faqs/farcaster/farcaster/farcaster-users-details#get-farcaster-profile-details-by-0x-address) (or something similar) to fetch Farcaster / Lens profile pic by 0x address.